tests/bsim/bluetooth/mesh/src/test_suspend.c

```diff
@@ -62,7 +71,22 @@ static int msg_handler(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx
 	prev_num = num;
 
 	/* Ensure that no message is received while Mesh is suspended or disabled. */
-	ASSERT_FALSE_MSG(suspended, "Received publication while Mesh is suspended.");
+	if (dut_status == DUT_SUSPENDED) {
+		if (num == (NUM_PUB + 1)) {
+			/* A publication may be sent just before DUT is suspended. Do nothing. */
```

Comment:
Strange alignment

Reply:
Strange how? It is an empty if statement?

Reply:
Why do you need empty `if` statement?

Reply:
Just one possible solution to make tester ignore a publication that might be sent before the DUT is fully suspended.